models/nxos/lldp/nxos_lldp_global.yml

```diff
@@ -0,0 +1,62 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: | 
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: nxos
+RESOURCE: lldp
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_lldp_global
+  version_added: 2.9
+  short_description: Configure and manage Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+  description: This module configures and manages the Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+  author: Adharsh Srivats Rangarajan (@adharshsrivatsr)
+  options:
+    config:
+      description:
+        - A list of link layer discovery configurations
+      type: list
+      elements: dict
+      suboptions:
```

Comment:
Add another parameter called `enabled` of type bool that enables or disables lldp.